docs/index.md

```diff
@@ -1,9 +1,23 @@
-# Welcome
+# Introduction to Percona Everest
```

Comment:
Can we call the menu link to the intro page "Home"?

Reply:
Also, if we list the benefits on the index page, will we still use the "[Why Everest?](https://everest-doc-pr-20.onrender.com/welcome/whyeverest.html)" topic?

Reply:
The structure would probably change. Catalina will structure it later.